cdap-ui/app/cdap/components/AppHeader/AppDrawer/AppDrawer.tsx

```diff
@@ -139,6 +140,17 @@ class AppDrawer extends React.PureComponent<IAppDrawerProps> {
             id="navbar-home"
             isActive={location.pathname === `/cdap/${nsurl}`}
           />
+          <ExperimentalFeature name={'data-ingestion'}>
+            <DrawerFeatureLink
+              featureName="Ingest"
+              featureUrl={`/${nsurl}/ingestion`}
+              featureSVGIconName="icon-ingest"
+              componentDidNavigate={componentDidNavigate}
+              data-cy="navbar-ingestion-link"
+              id="navbar-ingestion"
+              isActive={location.pathname === `/cdap/${nsurl}/ingestion`}
```

Comment:
i don't think this is needed if there's only one path

Reply:
done.